### PR TITLE
[#165785568] implement feature to enable super user to grant access to users to be staff

### PR DESCRIPTION
**Pull request by OnyekachiSamuel**
#### What does this PR do?
* Gives superuser the ability to make a user staff in the ART app
#### Description of Task to be completed?
* Create EditableUserDetailsComponent component
* Add actions and reducers
* Add test for EditableUserDetailsComponent component
* Add test for actions
* Add test for reducers
#### What are the relevant pivotal tracker stories?
[Pivotal tracker task Link](https://www.pivotaltracker.com/story/show/165785568)
#### Packages installed
None

### CODE REVIEW

File: src/_css/UserDetailComponent.scss [hunk 1]
```diff
@@ -22,3 +89,6 @@ h3.ui.header {
   width: 90%;
   align-content: center;
 }
+
+
```
Comment (fojuri) [inline src/_css/UserDetailComponent.scss]: Please remove the 2 extra spaces

**Review by fojuri — COMMENTED** [review]
Fix the console error when the User Detail component is rendered.

File: src/components/User/UserDetailComponent.jsx [hunk 2]
```diff
@@ -2,18 +2,20 @@ import React from 'react';
 import { isEmpty } from 'lodash';
 import PropTypes from 'prop-types';
 import { Header, Icon, Table, Segment, Card } from 'semantic-ui-react';
-import LoaderComponent from '../../components/LoaderComponent';
+// import LoaderComponent from '../../components/LoaderComponent';
```
Comment (fojuri) [inline src/components/User/UserDetailComponent.jsx]: Remove commented line

File: src/components/User/UserDetailComponent.jsx [hunk 3]
```diff
@@ -22,65 +24,50 @@ const assetsAssigned = (allocatedAssets) => {
...
   }
 
   return (
     <div>
       <Header as="h2" textAlign="left">
         {props.userDetail.full_name || 'Andelan'}
       </Header>
+      {
+        (props.successMessage !== '' || props.errorMessage !== '') && (
+          <StatusMessageComponent
```
Comment (fojuri) [inline src/components/User/UserDetailComponent.jsx:57]: The success Message only displays once and never displays if another action is dispatched. Please fix this.

File: src/_css/UserDetailComponent.scss (hunk 1 above)
Comment (fojuri) [reply src/_css/UserDetailComponent.scss]: You have not removed the extra spaces. You can configure your IDE to automatically remove trailing white spaces

**Review by fojuri — COMMENTED** [review]
I can still see the error on the console. Also attend to the failing tests.

File: src/components/User/UserDetailComponent.jsx (hunk 3 above)
Comment (OnyekachiSamuel) [reply src/components/User/UserDetailComponent.jsx:57]: Fixed now 😁

**Review by fojuri — APPROVED** [review]

**Pull request merged**